ppl/zqd/db/db.go

```diff
@@ -23,6 +24,24 @@ type DB interface {
 
 type DBKind string
 
+// Implement the flag.Value interface for type DBKind
```

Comment:
I suggest either removing the comment, or moving & adjusting it for godoc formatting (`Set implements ...`).